app/models/setting/remote_execution.rb

```diff
@@ -22,7 +22,10 @@ def self.load_defaults
                  'root'),
         self.set('remote_execution_effective_user_method',
                  N_('What command should be used to switch to the effective user. One of %s') % SSHExecutionProvider::EFFECTIVE_USER_METHODS.inspect,
-                 'sudo'),
+                 'sudo',
+                 'remote_execution_effective_user_method',
+                 nil,
+                 { :collection => Hash[SSHExecutionProvider::EFFECTIVE_USER_METHODS.map{|method| [method, method]}]}),
```

Comment:
This needs to be a proc, as Foreman changed how this worked shortly after, see https://github.com/theforeman/foreman/commit/1f823285f5c8231fc6afab0c7c2064690fde0d39